packages/nextcloud/test/core_test.dart

```diff
@@ -1,160 +1,162 @@
-      final navigationApps = await client.core.navigation.getAppsNavigation();
-      expect(navigationApps.ocs.data, hasLength(6));
-      expect(navigationApps.ocs.data[0].id, 'dashboard');
-      expect(navigationApps.ocs.data[1].id, 'files');
-      expect(navigationApps.ocs.data[2].id, 'photos');
-      expect(navigationApps.ocs.data[3].id, 'activity');
-      expect(navigationApps.ocs.data[4].id, 'notes');
-      expect(navigationApps.ocs.data[5].id, 'news');
-    });
-
-    test(
-      'Autocomplete',
-      () async {
-        final response = await client.core.autoComplete.$get(
-          search: '',
-          itemType: 'call',
-          itemId: 'new',
-          shareTypes: [
-            ShareType.user.index,
-            ShareType.group.index,
-          ],
-        );
-        expect(response.ocs.data, hasLength(3));
-
-        expect(response.ocs.data[0].id, 'admin');
-        expect(response.ocs.data[0].label, 'admin');
-        expect(response.ocs.data[0].icon, 'icon-user');
-        expect(response.ocs.data[0].source, 'users');
-        expect(response.ocs.data[0].status, isEmpty);
-        expect(response.ocs.data[0].subline, '');
-        expect(response.ocs.data[0].shareWithDisplayNameUnique, 'admin@example.com');
-
-        expect(response.ocs.data[1].id, 'user2');
-        expect(response.ocs.data[1].label, 'User Two');
-        expect(response.ocs.data[1].icon, 'icon-user');
-        expect(response.ocs.data[1].source, 'users');
-        expect(response.ocs.data[1].status, isEmpty);
-        expect(response.ocs.data[1].subline, '');
-        expect(response.ocs.data[1].shareWithDisplayNameUnique, 'user2');
-
-        expect(response.ocs.data[2].id, 'admin');
-        expect(response.ocs.data[2].label, 'admin');
-        expect(response.ocs.data[2].icon, '');
-        expect(response.ocs.data[2].source, 'groups');
-        expect(response.ocs.data[2].status, isEmpty);
-        expect(response.ocs.data[2].subline, '');
-        expect(response.ocs.data[2].shareWithDisplayNameUnique, '');
-      },
-      skip: true, // TODO: This test only works on 28+ due to a bug fix with the status
-    );
-
-    test('Get preview', () async {
-      final response = await client.core.preview.getPreview(file: 'Nextcloud.png');
-      expect(response, isNotEmpty);
-    });
-
-    test('Get avatar', () async {
-      final response = await client.core.avatar.getAvatar(userId: 'admin', size: 32);
-      expect(response.data, isNotEmpty);
-    });
-
-    test('Get dark avatar', () async {
-      final response = await client.core.avatar.getAvatarDark(userId: 'admin', size: 32);
-      expect(response.data, isNotEmpty);
-    });
-
-    test('Delete app password', () async {
-      await client.core.appPassword.deleteAppPassword();
-      expect(
-        () => client.core.appPassword.deleteAppPassword(),
-        throwsA(predicate((final e) => (e! as DynamiteApiException).statusCode == 401)),
+  group(
+    'core',
+    () {
```

Comment:
I don't like this formatting change. Doesn't seem necessary to me?

Reply:
Ah I see what you did. I don't like it, but I guess we'll have to live with it.

Reply:
We could also specify the timeout at the lib level (similar to what we did with the retry before) but I don't really like having them global.

The new uri test for example doesn't need a retry at all.